src/Services/WorkItemMutationWorkflow.php

```
@@ -0,0 +1,100 @@
+
+        return ['project_id' => $projectId, 'requires_review' => $requiresReview];
+    }
+
+    public function close(int $itemId, int $orgId): void
+    {
+        HLWorkItem::updateForOrg($this->db, $itemId, $orgId, ['status' => 'closed']);
+    }
+
+    public function deleteAndRenumber(int $itemId, int $orgId, int $projectId): int
+    {
+        HLWorkItem::deleteForOrg($this->db, $itemId, $orgId);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Models\HLWorkItem' in method 'deleteAndRenumber'. (undefined)

(StaticAccess)